### ISSUE TITLE
New generation sets Mumbai details to null

### ISSUE THREAD

**Issue by renatodeleao** [tobilg/aws-edge-locations#4]
Hey again!
Forked the project to play around (and give a hand debugging/contributing). As of today, when doing a new generation, new cities are added but Mumbai details are reseted to null/0 values. There's a lot of false positives for Mumbai at [the dataset, (someone had fun polluting the dataset)](https://datahub.io/core/airport-codes/r/airport-codes.json), but the reason is the "International" if condition at `generate.js`

https://github.com/tobilg/aws-edge-locations/blob/d913934ef6fbe358c36f14ebd699dff1120a69c5/generate.js#L122-L131

As you certainly know, in javascript, `-1` is considered a [truthy value](https://developer.mozilla.org/en-US/docs/Glossary/Truthy) so that condition is always truth (unless "international" sits at index 0 of the string) causing `tempMatches` to be (most of the times) equal to `matches`.

```diff
- if (m.name.toLowerCase().indexOf('international')) {
+ if (m.name.toLowerCase().indexOf('international') !== -1) {
  tempMatches.push(m);
}
```

Fixing this, revealed another error in the next condition

```js
if (tempMatches.length > 0) { // Multiple matches, take first one, kind of random selection 
  match = tempMatches[0]; 
} 
// but what if tmpMatches is empty? as in no "international" thing on any
```

At least for now i think we should fallback to the first match (as we do if we only get a single match)

```diff
if (tempMatches.length > 0) { // Multiple matches, take first one, kind of random selection
  match = tempMatches[0];
+ } else {
+  match = matches[0] // no tempMatches (as in "no internationals") also defaults to first occurrence of matches
+ }
```

As you've stated in your comments, this can be improved with some refined regexp wizardry, but i think we can patch it like this for now. Thoughts? 
If you want i can submit a PR for this.

**Comment by tobilg** [comment]
Hi @renatodeleao, thanks again for submitting the issue. I created a PR (#5 ) from your fork, I assume this will include the fix you were mentioning?

**Comment by tobilg** [comment]
This should be fixed in version `0.2.0`.

**Comment by renatodeleao** [comment]
Sry for the late reply, but thanks for the release!
